lib/get-dialect-ids.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from "vitest";
+import { getDialectIds, loadDialect } from "./keywords.js";
+import "../draft-2020-12";
+import "../draft-2019-09";
+
+
+test("should return only imported schema in the array array if no custom dialects are loaded", () => {
+  const dialectIds = getDialectIds();
+  expect(dialectIds).toEqual(["https://json-schema.org/draft/2020-12/schema"]);
+});
+
+test("should return an array of dialect identifiers", () => {
```

Comment:
This test description needs to be better. You describe well what the expectation is, but not under what circumstances you expect that result. Hint: Say something about this testing the custom dialect case.